### DISCUSSION TITLE
Can oh-my-bash and magit coexist in .bashrc

### DISCUSSION THREAD

**Discussion by OSStuntman** [ohmybash/oh-my-bash#558] [Q&A]
Hello, 

Can oh-my-bash and magit coexist in .bashrc? How do I merge the two .bashrc?

**Comment by akinomyoga** [comment]
Could you explain what is "magit" and how you would configure magit in .bashrc without Oh My Bash? There seems to be an Emacs package [`magit`](https://github.com/magit/magit), but I'm not sure if this is what you mean by magit. It doesn't seem to have a specific setting in `.bashrc` [as far as I search in the repository](https://github.com/search?q=repo%3Amagit%2Fmagit+bashrc&type=issues).

**Reply by OSStuntman** [reply]
Yes, that's correct it's a Emacs mode for Git. 

After Oh-my-bash was installed, when I pull up the Magit status buffer it's empty: Magit hasn't noticed any of the files I've modified or added to the directory.

.bashrc + ohmybash: https://drive.proton.me/urls/DECEAM45WG#3FcYhodiydF9
.bashrc before ohmybash:  https://drive.proton.me/urls/TSFJD3GPR8#N9KhHiRhanKl

**Reply by akinomyoga** [reply]
You seemed to have requested the way to make OMB and Magit *coexist* in .bashrc, but I don't seem to find any Magit setting in your second `.bashrc` (".bashrc before ohmybash"). So, "*the Magit setting does not actually exist in `.bashrc`, but Magit worked with an existing .bashrc without specific settings for Magit. Magit just doesn't work with the OMB .bashrc for some reason.*" Is that your situation?
